jupyterlab_gitplus/github_v3.py

```diff
@@ -28,7 +28,12 @@ def create_pull_request(owner_login, repo_name, title, head, base, access_token)
         response.raise_for_status()
         result = {
             'github_url': content['html_url'],
-            'reviewnb_url': content['html_url'].replace('github.com', 'app.reviewnb.com')
```

Comment:
(after following my top comment above) this would just become - 

```
'reviewnb_url': content['html_url'].replace('https://github.com/', reviewnb_endpoint)
```